core/src/test/java/com/sequenceiq/cloudbreak/converter/stack/instance/InstanceGroupToInstanceGroupResponseConverterTest.java

```diff
@@ -48,14 +49,15 @@ public void testConvert() {
 
         assertEquals(source.getNodeCount(), result.getNodeCount());
         assertEquals(getSource().getInstanceGroupType(), result.getType());
-        assertAllFieldsNotNull(result, Lists.newArrayList("recipes", "template", "securityGroup", "gcp", "name", "mock", "openstack",
+        assertAllFieldsNotNull(result, Lists.newArrayList("recipes", "template", "network",
+                "securityGroup", "gcp", "name", "mock", "openstack",
                 "aws", "yarn", "azure"));
     }
 
     @Override
     public InstanceGroup createSource() {
-        var instanceGroup = TestUtil.instanceGroup(1L, InstanceGroupType.CORE, TestUtil.gcpTemplate(1L));
-        var metaData = TestUtil.instanceMetaData(1L, 1L, InstanceStatus.SERVICES_RUNNING, false, instanceGroup);
+        InstanceGroup instanceGroup = TestUtil.instanceGroup(1L, InstanceGroupType.CORE, TestUtil.gcpTemplate(1L));
```

Comment:
I support eliminating `var` in favor of explicit types, so perhaps you could do the same to the remaining `var` occurrences as well. 😄